services/docker-stack-as-ap-opticscorr.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  tb-ps:
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:2020-02-26
+    command: bash -c 'source /etc/profile && /ioc-apps/as-ap-opticscorr.bash'
+    deploy:
+      placement:
+        constraints:
+          - node.hostname != CA-RaTim-CO-IOCSrv
```

Comment:
This server is responsible for the timing IOCs.
Is this a typo, or we are actively avoiding it for some reason?

Reply:
it is not a type. I understand from talking to you guys we wanted to run fac services only in dell nodes, no? can we include CA-RaTim-CO-IOCSrv ?

Reply:
CA-RaTim-CO-IOCSrv IS a dell server. issue understood. I will comment this constraint.